src/Raven.Client/Documents/DatabaseTopology.cs

```diff
@@ -19,6 +19,26 @@ public bool RelevantFor(string nodeTag)
                    Watchers.Contains(nodeTag);
         }
 
+        public IEnumerable<string> AllNodes
```

Comment:
Should we include this? We don't want to failover to promotables / watcher nodes, do we?